### ISSUE TITLE
"text-align" CSS property should also align images in table cells

### ISSUE THREAD

**Issue by hakimio** [dompdf/dompdf#1420]
If you add some fixed size images using "img" tag to table cells and set table cell "text-align" CSS property, it will not only align table cell text but also align the images when html is rendered in all major browsers. However, when the same html is rendered with dompdf, all images in table cells will not be effected by "text-align".

**Comment by bsweeney** [comment]
Do you have a sample that exhibits the issue? I tried the following and it looked ok:

```
<!DOCTYPE html>
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
</head>
<body>
  <table>
    <tr>
      <td style="text-align: right;">
        <img src="http://eclecticgeek.com/images/macro/ants.jpg" width="100">
        <br>
        aasdfasd fasdf asdf asdf  adsf asdfaf 
      </td>
    </tr>
  </table>
</body>
</html>
```

**Comment by hakimio** [comment]
Can you test instead of in-lining the style to put sth like the following:
<style>
.ttext td {
        text-align: center;
    }
</style>

<table>
    <tr class="ttext">
      <td>
        <img src="http://eclecticgeek.com/images/macro/ants.jpg" width="100">
        <br>
        aasdfasd fasdf asdf asdf  adsf asdfaf 
      </td>
    </tr>
  </table>

**Comment by hakimio** [comment]
Try the following:
```
<style>
.ttext td {
        text-align: center;
    }
.icon {
	width: 14px;
	height: 14px;
	display: inline-block;
	background-size: 14px;
	background-image: url(http://eclecticgeek.com/images/macro/ants.jpg);
}
</style>

<table>
    <tr class="ttext">
      <td>
        <div class="icon"></div>
        <br>
        aasdfasd fasdf asdf asdf  adsf asdfaf 
      </td>
    </tr>
</table>
```

**Comment by bsweeney** [comment]
I see. Looks to be more of a problem with inline-blocks than with images.

**Comment by Mellthas** [comment]
This is fixed by #2505.
